core/src/main/resources/error/error-classes.json

```diff
@@ -63,5 +63,86 @@
   "WRITING_JOB_ABORTED" : {
     "message" : [ "Writing job aborted" ],
     "sqlState" : "40000"
+  },
+  "INVALID_INSERT_INTO_CONTEXT" : {
+    "message" : [ "Invalid InsertIntoContext" ],
+    "sqlState" : "10001"
+  },
+  "INSERT_OVERWRITE_DIRECTORY_UNSUPPORTED" : {
+    "message" : [ "INSERT OVERWRITE DIRECTORY is not supported" ],
+    "sqlState" : "10002"
+  },
+  "COLUMNS_ALIASES_NOT_ALLOWED_IN_OPERATION" : {
+    "message" : [ "Columns aliases are not allowed in %s." ],
+    "sqlState" : "10003"
+  },
+  "EMPTY_SOURCE_FOR_MERGE" : {
+    "message" : [ "Empty source for merge: you should specify a source table/subquery in merge." ],
+    "sqlState" : "10004"
+  },
+  "UNRECOGNIZED_MATCHED_ACTION" : {
```

Comment:
Can we simplify this by parametrizing? Then we only need one for `MATCHED`/`NOT_MATCHED`.

Reply:
OK